tools/migrate_issues_gh.py

```diff
@@ -51,6 +51,10 @@
 from launchpadlib.launchpad import (
```

Comment:
we don't fix this module, it's here only for historical reasons (can be deleted in a "delete obsolete code" PR